product_supplierinfo_import/readme/DESCRIPTION.md

```diff
@@ -1,2 +1 @@
-This module allows to import vendor price lists from excel files matching the products
-from their barcode.
+This module allows to import vendor price lists from excel.
```

Comment:
Don't drop the whole matching thing. Just change to state that the matching can be done by any template char field